powers.c

```diff
@@ -0,0 +1,19 @@
+/*Rebecca Hale, Assignment 2 Part 1*/
```

Comment:
Nice job! The only strange thing is that it prints Enter a number three times when input is incorrectly given.